Contribute/how-to-write-use-markdown.md

```diff
@@ -362,9 +425,7 @@ Here are the encodings for the "smart" versions of these punctuation marks:
 
 ### Angle brackets
 
-It is common to use angle brackets to denote a placeholder. When you do this in text (not code),
-you must encode the angle brackets. Otherwise, Markdown thinks that they're intended to be an HTML
-tag.
+It is common to use angle brackets to denote a placeholder. When you do this in text (not code), you must encode the angle brackets. Otherwise, Markdown thinks that they're intended to be an HTML tag.
```

Comment:
Is escaping angle brackets acceptable? I often see them escaped with a backslash rather than HTML encoded.